### DISCUSSION TITLE
Support for launching notebooks in Deepnote

### DISCUSSION THREAD

**Discussion by jakubzitny** [executablebooks/meta#468] [Ideas]
Hi there, I'd like to submit a PR with adding support for launching notebooks in [Deepnote](https://deepnote.com) as an alternative to Google Colab and Binder. Is this something that this community would welcome?

Also, I was trying hard to find the best way to implement this in the jupyter-book repo, but I am not sure the main logic is implemented there. So feel free to point me in the right direction as well 🙏 😇

**Comment by chrisjsewell** [comment]
Heya, yeh looks interesting cheers. This functionality would likely reside in https://github.com/executablebooks/sphinx-book-theme

**Reply by chrisjsewell** [reply]
This is where the launch buttons are added: https://github.com/executablebooks/sphinx-book-theme/blob/master/sphinx_book_theme/topbar/launchbuttons.html

**Reply by jakubzitny** [reply]
Wow, thanks for pointing it out, my search game was weak when searching for it. Anyway, I submitted two draft PRs:
- https://github.com/executablebooks/sphinx-book-theme/pull/385
- https://github.com/executablebooks/jupyter-book/pull/1434

I am still working on configuring the whole dev env locally, so once I test everything I'll make the PRs ready for review. Feedback is welcome 🙂

**Reply by chrisjsewell** [reply]
Cheers!
